deploy/docs/Security_Best_Practices.md

```diff
@@ -241,41 +241,147 @@ spec:
 You should only leave the ports for the protocol you're actually using to deliver spans to the receiver in the above definition. As with Prometheus,
 the above configuration is very permissive for ingress. You're encouraged to make it more restrictive based on your specific requirements.
 
-## Hardening fluentd StatefulSet with `securityContext`
+## Setting `securityContext`
 
 One can use `fluentd.securityContext` and
 `fluentd.(logs|metrics|events).statefulset.containers.fluentd.securityContext`
 to tighten up the security requirements for fluentd containers running as part
 of collection StatefulSets.
 
+`securityContext` for FluentBit can set using `fluent-bit.securityContext` or `fluent-bit.podSecurityContext`.
+
 One example of such a configuration can be found below:
 
 ```yaml
-fluentd:
-  ...
+fluent-bit:
   securityContext:
-    ## The group ID of all processes in the statefulset containers.
-    ## By default this needs to be fluent(999).
-    fsGroup: 999
-    runAsNonRoot: true
+    readOnlyRootFilesystem: true
+    runAsUser: 1000
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+      - ALL
+  initContainers:
+  - name: changeowner
+    image: public.ecr.aws/docker/library/busybox
+    command:
+    - "sh"
+    - "-c"
+    - chown -R 1000 /tail-db
+    volumeMounts:
+    - mountPath: /tail-db
+      name: tail-db
+    securityContext:
+      readOnlyRootFilesystem: true
+      runAsUser: 1000
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
```

Comment:
Are you sure this will work correctly on all K8s distributions? I recall `/var/log/pods` having different ownership between them.